spec/lib/goodcity/organisation_populator_spec.rb

```diff
@@ -4,10 +4,12 @@
 describe Goodcity::OrganisationPopulator do
   let(:organisation_populator) { Goodcity::OrganisationPopulator.new }
   let(:file)                   { File.read("#{Rails.root}/spec/fixtures/organisation.json")}
+  let!(:country)               { FactoryGirl.create(:country, name_en: "China - Hong Kong (Special Administrative Region)") }
```

Comment:
@trojanh in general, it's better to use factories to create data and also `let` syntax. (although let! vs before is debatable!)